templates/htmx-adventure-card.html

```diff
@@ -57,7 +57,7 @@
 
         {% if adventure["code"] %}
             <p class="text-gray-600 mt-1"><strong>{{_('number_lines')}}:</strong> {{ adventure.number_lines }}</p>
-            <pre level="{{ adventure.level }}">{{ adventure.code }}</pre>
+            <pre level="{{ adventure.level }}" class="text-xs">{{ adventure.code }}</pre>
```

Comment:
To my eyes the editor content is really small now! Any reason for that?!

Reply:
To my eyes they were very large, not showing a lot of the program because of it.

Meet in the middle at `text-sm` ? 😜

Reply:
Hahaha, amazing!!